openquake/server/views.py

```diff
@@ -334,7 +336,6 @@ def calc_remove(request, calc_id):
         return HttpResponse(content=message,
                             content_type='text/plain', status=500)
```

Comment:
But did you read my comment and read my commit here? https://github.com/gem/oq-engine/commit/85c547647e6c0f6c5d23dc493894202a89c121e7 
There is error 403 and not 500